tests/gdimagesetinterpolationmethod/github_bug_00847.c

```diff
@@ -0,0 +1,20 @@
+#include "gd.h"
+#include "gdtest.h"
+
+int main()
+{
+  gdImagePtr im;
+  int status;
+  im = gdImageCreate(1, 1);
+  gdImageSetInterpolationMethod(im, GD_BICUBIC_FIXED);
+  if (gdImageGetInterpolationMethod(im) == GD_BICUBIC_FIXED) {
+    status = 0;
+  } else {
+    status = 1;
+  }
+  if (status > 0) {
+    gdTestErrorMsg("expecting success return value\n");
+  }
+  gdImageDestroy(im);
+  return status;
```

Comment:
For the above modifications to have effect, we should do:
```suggestion
  return gdNumFailures();
```